src/components/MovieListPage/MovieListPage.jsx

```diff
@@ -12,68 +13,122 @@ import "../MovieForm/style.css";
 import "../Dialog/style.css";
 import Dialog from "../Dialog/Dialog";
 import MovieForm from "../MovieForm/MovieForm";
-
+import AddMovie from "../Dialog/AddMovie";
+import MovieDetails from "../MovieDetails/MovieDetails";
+import axios from "axios";
 
 function MovieListPage() {
-  const [querySearch, setQuerySearch] = useState("Search film here");
+  const [querySearch, setQuerySearch] = useState("");
   const [sortOrder, setSortOrder] = useState("");
-  const [genre, setGenre] = useState(1);
+  const [genre, setGenre] = useState("All");
   const [genres, setGenres] = useState([
```

Comment:
You can use const for this variable if you don't get it from BE. The useState is overkill here.